backend/plugin/mfa/otp/otp.go

```diff
@@ -2,87 +2,26 @@
 package otp
```

Comment:
In this case, we don't need a plugin any more. This can be part of util.go or inline with the caller code.